.github/workflows/blackduck_scan.yml

```diff
@@ -0,0 +1,49 @@
+name: Black Duck Scan
+
+on:
+  workflow_dispatch:
+    inputs:
+      production:
+        description: Production
+        type: boolean
+        default: true
+
+jobs:
+  scan-code:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+        with:
+          # Number of commits to fetch. 0 indicates all history for all branches and tags.
+          # Pulls all commits (needed for Lerna)
+          fetch-depth: 0
+      - name: Install Dependencies
+        run: npm ci --ignore-scripts --audit=false --fund=false
+      - name: Scanning
+        run: |
+          DETECT_FILE="synopsys.jar"
+          curl https://sig-repo.synopsys.com/artifactory/bds-integrations-release/com/synopsys/integration/synopsys-detect/${{ vars.SYNOPSYS_DETECT_VERSION }}/synopsys-detect-${{ vars.SYNOPSYS_DETECT_VERSION }}.jar -o $DETECT_FILE
+          mkdir appsec || true
+          mv $DETECT_FILE appsec/
+
+          if [[ "${{ inputs.production }}" == "true" ]]; then  export RELEASE_TYPE="PROD"; export VERSION_ID="Release_${GITHUB_REF#refs/heads/}"; fi
```

Comment:
Is It possible to move this line into env tag ? Somthing like this.
`env:
      RELEASE_TYPE: ${{ if inputs.production == 'true' }}PROD${{ else }}DEV${{ endif }}
      VERSION_ID: ${{ if inputs.production == 'true' }}Release_${{ github.ref#refs/heads/ }}${{ else }}Development-SINGLE${{ endif }}`

Reply:
It's not possible